Wwise that works

tomy204 opened the pull request:
with the addons files addes to the gitignore

should be able to install by:

fetching from this branch/main once merged and pull
-run the integration app from here https://github.com/alessandrofama/wwise-godot-integration/releases/tag/1.5.0_Wwise2021.1.4.7707
-if there are cahnged that appear on githubdesktop discard them

mikezimean committed:
Godot project with Wwise integration settings

I pray to all kinds of deities and demons it works now

tomy204 committed:
Update .gitignore

mikezimean committed:
A little checkbox in Project Settings

Prayers obviously didn't work as planned

Maaack committed:
Adds the wise integration addon to the repo.

Maaack committed:
Activate the integration.

Maaack committed:
Merge branch 'Wwise-that-Works' into integrating-wise-app-2

# Conflicts:
#	project.godot

Maaack reviewed (approved):
Looks good! I added the wwise addon to the repo, too, as that was only 175 kb. It helped keep things in sync with individual's project.godot file. Programmers can then download the >100mb builds for their individual system into the /bin/ folder of that addon.

Maaack committed:
Update the readme to include instructions for the Wwise integration.

Maaack committed:
Update README, make instructions verbose.

amythyst221 committed:
WwiseShaderTest

Created new WwiseShaderTest that's a copy of the ShaderTest scene and removed Wwise elements from ShaderTest. Made WwiseShaderTest the default

Maaack committed:
Update README.

Maaack committed:
Merge remote-tracking branch 'origin/Wwise-that-Works' into Wwise-that-Works
